invenio_vocabularies/contrib/subjects/schema.py

```diff
@@ -50,4 +64,5 @@ class SubjectRelationSchema(ContribVocabularyRelationSchema):
     subject = SanitizedUnicode()
     scheme = SanitizedUnicode()
     title = i18n_strings
+    props = fields.Dict()
     synonyms = fields.List(SanitizedUnicode())
```

Comment:
minor: all these should be `dump_only` fields. Only `subject` is acceptable as input (i.e. when we don't "link" to a subject via its `id`)